plenum/common/messages/internal_messages.py

```diff
@@ -1,13 +1,13 @@
 from typing import NamedTuple, List, Any
 
-from plenum.common.messages.node_messages import CheckpointState
+from plenum.common.exceptions import SuspiciousNode
+from plenum.common.messages.node_messages import CheckpointState, PrePrepare
 
+# TODO: should be removed
 ValidatorsChanged = NamedTuple('ValidatorsChange',
```

Comment:
If this should be removed then how services would become aware of validator set update?